omnipaxos/src/util.rs

```diff
@@ -401,3 +401,51 @@ impl Quorum {
         }
     }
 }
+
+/// Structs for the UI
+pub mod ui {
```

Comment:
This file is supposed to be util functions and structs related to consensus logic. Other utils should be in the `utils` mod, so move this whole module to new file in utils/ui.rs